packages/logging/logging.js

```diff
@@ -84,43 +88,46 @@ var logInBrowser = function (obj) {
 };
 
 // @returns {Object: { line: Number, file: String }}
-Log._getCallerDetails = function () {
-  var getStack = function () {
+Log._getCallerDetails = () => {
+  const getStack = () => {
     // We do NOT use Error.prepareStackTrace here (a V8 extension that gets us a
     // pre-parsed stack) since it's impossible to compose it with the use of
     // Error.prepareStackTrace used on the server for source maps.
-    var err = new Error;
-    var stack = err.stack;
+    const err = new Error;
+    const stack = err.stack;
     return stack;
   };
 
-  var stack = getStack();
+  const stack = getStack();
 
-  if (!stack) return {};
-
-  var lines = stack.split('\n');
+  if (!stack) {
+    return {};
+  }
 
   // looking for the first line outside the logging package (or an
   // eval if we find that first)
-  var line;
-  for (var i = 1; i < lines.length; ++i) {
-    line = lines[i];
+  let line;
+  const [_, ...lines] = stack.split('\n');
```

Comment:
The use of `_` here threw me off for a second when reviewing. Since this is intended to hold the ignored first line, maybe we should just call it `firstLine` or `ignoredFirstLine`.

Reply:
@hwillson in more functional languages like haskell underscore is used as a kind of "don't care" symbol. But I'll change it since in javascript it's a bit weird since the underscore symbol is already used for lots of other things. I'll just use a slice, so we're not making variables that are not getting used.

Reply:
Awesome, thanks @sebakerckhof! I have a Haskell book sitting beside me that I've been planning on cracking open for 8 months now ... *sigh*.

Reply:
For what it's worth, you can leave out the `_` entirely, and the first element will be thrown away.  Take that, functional 💥:

```js
const [ , ...lines] = stack.split('\n');
```